client/my-sites/plugins/plans/style.scss

```diff
@@ -6,3 +6,17 @@
 		font-size: $font-body;
 	}
 }
+
+.plugins-plans {
+	@include breakpoint-deprecated( "<660px" ) {
+		margin: 0 24px !important;
+
+		.formatted-header {
+			display: none;
+		}
+
+		.plan-features__table {
+			display: none !important;
+		}
```

Comment:
Plans table already does this under /start - why do we need this here?

![Screenshot 2022-09-21 at 14-57-47 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/191417771-56c78945-24cc-4ff0-8e0c-1c6949914924.png)